x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/entsearch/engine/EngineIndexService.java

```diff
@@ -177,13 +186,68 @@ public void getEngine(String resourceName, ActionListener<Engine> listener) {
         }));
     }
 
+    private static String getEngineAliasName(Engine engine) {
```

Comment:
This could be part of the Engine class, but I feel it's strongly related to how this service works and not the data that is held by the Engine.

Reply:
It's two methods as of now - let's discuss this when we add the delete part 👍